src/components/index.js

```diff
@@ -132,6 +134,17 @@ class IProve extends Component {
     }
   }
 
+  updateDependenciesFromInsertionAndRemoval = (index, increment) => {
+    for (var i = 0; i < this.props.steps.length; i++) {
+      for (var j = 0; j < this.props.steps[i].dependencies.length; j++) {
```

Comment:
don't use var as its the old javascript syntax - use let or const instead
in this case try using the foreach loop